inventory/any.cf

```diff
@@ -555,13 +555,17 @@ bundle agent cfe_autorun_inventory_packages
                                  "0");
 
   packages:
-    debian::
+    # In 3.7 we are usining package_module specific way of generating software inventory.
+    # In this case installing non_existing_package is not needed any more.
+    !cfengine_3_7.debian::
```

Comment:
This should be inclusive rather than exclusive, in order to be future proof. I would also rephrase the sentence a bit. Maybe:

```
# From 3.7 onwards there is a new package promise implementation using package
# modules, in which case installing non_existing_package is not needed any more.
(cfengine_3_5|cfengine_3_6).debian::
```

And similar for RedHat.